src/js/components/header/index.js

```diff
@@ -0,0 +1,22 @@
+var yo = require('yo-yo')
+var header = require('dat-header')
+
+module.exports = Header
+
+function Header (el, props) {
+  if (!(this instanceof Header)) return new Header(el, props)
+  this.$el = document.getElementById(el)
+  this.props = props
+  this._component = this._render()
+  if (this.$el) this.$el.appendChild(this._component)
+}
+
+Header.prototype.update = function (state) {
+  yo.update(this._component, this._render())
```

Comment:
you should filter for the exact reducer you want to watch for state changes, otherwise this will get noisy and reduce performance.

see:
https://github.com/datproject/dat.land/blob/master/src/js/components/hyperdrive-size/index.js#L16-L18